src/adapter/src/catalog/builtin.rs

```diff
@@ -1567,13 +1567,17 @@ pub static MZ_CLUSTER_REPLICA_METRICS: Lazy<BuiltinTable> = Lazy::new(|| Builtin
         .with_column("memory_bytes", ScalarType::UInt64.nullable(true)),
 });
 
-pub static MZ_STORAGE_HOST_METRICS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
-    name: "mz_storage_host_metrics",
+pub static MZ_STORAGE_REPLICA_METRICS: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSource {
+    name: "mz_storage_replica_metrics",
     // TODO[btv] - make this public once we work out whether and how to fuse it with
     // the corresponding Compute tables.
     schema: MZ_INTERNAL_SCHEMA,
+    data_source: Some(IntrospectionType::StorageReplicaMetrics),
     desc: RelationDesc::empty()
-        .with_column("object_id", ScalarType::String.nullable(false))
+        // Right now, for each source/sink there is only one replica, which is
+        // identified by the global ID of the source/sink.
+        .with_column("replica_id", ScalarType::String.nullable(false))
```

Comment:
im a little bit curious about backwards compatibility here; I imagine when we move to multiple-storage's-per-storaged, for old-style sources that are their own cluster, we will leave this as the source/sink id, but for new-style ones, we will make it the replica global id?

Reply:
seems like we'll scrap this whole table then